tests/e2e/expected_results/maven/maven.json

```diff
@@ -116,7 +116,7 @@
                         "Build as code - SLSA Level 3"
                     ],
                     "justification": [
-                        "The target repository uses build tool maven in jenkins to deploy."
+                        "The target repository uses build tool maven in jenkins using artifactsPublisher to deploy."
```

Comment:
Is this something we expected to change? (i.e the addition of `using artifactsPublisher`)

Reply:
Yes, I think the report was not updated in the past because we don't really check the justification in e2e tests.